src/components/Calls/CallIFrame.js

```diff
@@ -6,12 +6,19 @@ import { createClassName } from '../helpers';
 import styles from './styles.scss';
 
 
-export const CallIframe = () => {
+export const CallIframe = (props) => {
 	const { token, room } = store.state;
 	const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }`;
+	const callInNewTab = async () => {
+		window.open(url);
+		await store.setState({ ongoingCall: { callStatus: 'ongoingCallInNewTab', time: props.time } });
+		await store.setState({ incomingCallAlert: { show: false, callProvider: props.callProvider } });
+	};
 	return (
-		<div className={createClassName(styles, 'call-iframe')}>
-			<iframe className={createClassName(styles, 'call-iframe__content')} allow='camera;microphone' src={url} />
+		<div>
+			{(window.innerWidth <= 800) && (window.innerHeight <= 630)
```

Comment:
I don't think its a good idea to implement this logic over here. This component is named `CallIFrame` so it should only be responsible for dealing with the ongoing call in the iframe - it shouldn't be responsible for redirecting the user to another page